<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>diarios</class>
 <widget class="QDialog" name="diarios" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edición / Alta de diarios</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Código:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CodigoLineEdit" >
       <property name="maximumSize" >
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength" >
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DescripcionLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="maxLength" >
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="BotonGuardar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Guardar</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonEliminar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Eliminar</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonCancelar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonSalir" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tabla" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotonSalir</sender>
   <signal>clicked()</signal>
   <receiver>diarios</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>372</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>420</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
